lib/auth/methods.go

```diff
@@ -209,7 +209,8 @@ func (s *Server) authenticateUser(ctx context.Context, req AuthenticateUserReque
 // if authentication is successful. In case the existing session ID is used to authenticate,
 // returns the existing session instead of creating a new one
 func (s *Server) AuthenticateWebUser(req AuthenticateUserRequest) (types.WebSession, error) {
-	authPref, err := s.GetAuthPreference()
+	ctx := context.TODO()
+	authPref, err := s.GetAuthPreference(ctx)
```

Comment:
The context should really come from the caller, so I think it's better to leave it as a `context.TODO` rather than making it seem like the server's `closeCtx` is the correct context, even if it's slightly better in the short term.

Reply:
Are there plans to add a `Context` parameter, or is that a future problem?

Reply:
From my understanding it's a future problem and `context.TODO` is the best thing to insert to signal that.

Reply:
Fair enough.